PerfTools/AllocMonitor/test/BuildFile.xml

```diff
@@ -23,4 +23,15 @@
     <flags CXXFLAGS="-O0"/>
   </bin>
   <test name="TestPerfToolsModuleAllocMonitor" command="runModuleAlloc.sh"/>
+
+  <!-- can't be part of the catch2 tests above because of ASAN exclusion -->
+  <bin file="test_intrusiveAllocMonitor.cc" name="testIntrusiveAllocMonitor">
```

Comment:
How can you be sure this actually worked?

Reply:
Would the change I just pushed (requiring the output contains `Vector fill: requested`) be sufficient?

I thought about checking the actual numbers as well, but I'd also like to avoid making the test to depend on the standard library and memory allocator behavior. I guess I _could_ check the output more thoroughly and that all reported quantities are non-zero, but that would take a little bit of effort.

Reply:
That would be sufficient for me.